src/drawing/supports.c

```diff
@@ -278,7 +278,7 @@ bool wooden_a_supports_paint_setup(int supportType, int special, int height, uin
 				hasSupports = true;
 			} else {
 				hasSupports = true;
-				if (!sub_98198C(imageId, 0, 0, bBox.length.x, bBox.length.y, bBox.length.z, z, bBox.offset.x, bBox.offset.y, bBox.offset.z + z, rotation)) {
+				if (sub_98198C(imageId, 0, 0, bBox.length.x, bBox.length.y, bBox.length.z, z, bBox.offset.x, bBox.offset.y, bBox.offset.z + z, rotation)) {
```

Comment:
I'm returning the inverse value of the flag, so I just assumed. Will check with the decompiler though.